tests/admin_scripts/tests.py

```diff
@@ -3328,3 +3328,19 @@ def test_no_suggestions(self):
         out, err = self.run_django_admin(args)
         self.assertNoOutput(out)
         self.assertNotInOutput(err, "Did you mean")
+
+
+class AdminScriptsColorizedHelp(AdminScriptTestCase):
+    @unittest.skipUnless(PY314, "argparse colorized help requires Python 3.14+")
+    def test_help_is_colorized_on_py314(self):
+        # Use a command that doesn't need settings.
+        with mock.patch.dict(os.environ, {}, clear=False):
+            out, err = self.run_django_admin(["startproject", "--help"])
```

Comment:
can we use just `--help` ?

Reply:
•  --help triggers ManagementUtility’s custom help path that prints main_help_text(), not argparse’s help.
•  The colorization we’re testing is specifically argparse’s colorized help introduced in Python 3.14.
•  To hit argparse’s help (and thus see ANSI color sequences), we need a subcommand’s help, e.g., startproject --help (or equivalently help startproject).